### PR TITLE
Added flag to plot electrode names

### PR DESCRIPTION
**Pull request by wmvanvliet**
Now it becomes easier to see which channel ends up where. Looks like the rotation of the easycaps are wrong.

### CODE REVIEW

File: mne/viz/montage.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
+import numpy as np
+
 """Functions to plot EEG sensor montages
 """
 
 
-def plot_montage(montage, scale_factor=1.5):
+def plot_montage(montage, scale_factor=1.5, draw_names=False):
```
Comment (agramfort) [inline mne/viz/montage.py]: we call it show_names in topomap plots. I would rename it to be consistent. Besides LGTM.

**Comment by agramfort** [comment]
lgtm besides a weird travis failure.

**Comment by dengemann** [comment]
@wmvanvliet thanks!

**Comment by dengemann** [comment]
@wmvanvliet thanks!!

**Pull request merged**